code/csharp/main-workshop/src/CdkWorkshop/CdkWorkshop.csproj

```diff
@@ -2,19 +2,21 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.APIGateway" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.DynamoDB" Version="1.17.1-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.17.1-devpreview" />
+    <PackageReference Include="Amazon.CDK" Version="1.117.*" />
+    <PackageReference Include="Amazon.CDK.AWS.APIGateway" Version="1.117.*" />
+    <PackageReference Include="Amazon.CDK.AWS.DynamoDB" Version="1.117.*" />
+    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.117.*" />
 
-    <PackageReference Include="Eladb.DynamoTableViewer" Version="3.0.6" />
+    <PackageReference Include="Eladb.DynamoTableViewer" Version="3.1.*" />
```

Comment:
See my comment on #152 where I had to use `Version="[0.1.*,3)"` by running `dotnet add package Eladb.DynamoTableViewer --version "[0.1.*,3)"` to not get

> error NU1107: Version conflict detected for Amazon.JSII.Runtime.  Install/reference Amazon.JSII.Runtime 1.31.0 directly to project CdkWorkshop to resolve this issue.

Is there a corresponding doc change to run `dotnet add package Amazon.Jsii.Runtime --version "1.31.*"` to get this entry in the `.csproj` file when running the workshop?

Reply:
I found that using your versioning of `[0.1.*,3)` enabled me to remove the 2 Jsii dependencies so I decided to implement your versioning instead.

There is one place in the workshop guide that instructs the user to add the package. [I updated that instruction here](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/268/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.yml&file-filters%5B%5D=dotfile#diff-b200b09578c1ea2aacde8f31ddf0e21af2d50a3db364d76ca4428f42dc4a2929R12)